stellar/operations.py

```diff
@@ -36,14 +36,9 @@ def copy_database(raw_conn, from_database, to_database):
     terminate_database_connections(raw_conn, from_database)
 
     if raw_conn.engine.dialect.name == 'postgresql':
-        raw_conn.execute(
-            '''
-                CREATE DATABASE "%s" WITH TEMPLATE "%s";
-            ''' %
-            (
-                to_database,
-                from_database
-            )
+        sqlalchemy_utils.functions.create_database(
+            '%s%s' % (raw_conn.engine.url, to_database),
+            template=from_database
```

Comment:
> Because it's open!

Fair point, I had not realized this, but I doubt that it is my 3 years old comment that is preventing this PR from being merged.

It looks like the last commit on this project is from 2 years ago, so this project seems a little dead now, pity :(